jsonrpc2/server.go

```diff
@@ -99,16 +99,15 @@ func (s *Server) RegisterMethod(rpcName string, receiver interface{}, methodName
 	return nil
 }
 
+// TODO: Add RegisterFunction?
+
 // Handle executes a request message against the server registry.
-func (s *Server) Handle(ctx context.Context, req *Message) *Message {
-	r := &Message{
-		Response: &Response{
-			Result: nullResult,
-		},
-		ID:      req.ID,
-		Version: Version,
+func (s *Server) Handle(ctx context.Context, req *Request) *Response {
```

Comment:
This is breaking the public API, but nobody is using it yet and it's moderately internal, sooo seems safe? :man_shrugging: